app/components/blacklight/document_component.rb

```diff
@@ -137,6 +138,12 @@ def before_render
       end
     end
 
+    def view_config
+      @view_config ||= presenter&.view_config || Blacklight::Configuration::ViewConfig.new
```

Comment:
Is there really a case where presenter is `nil`?  Wouldn't we also need to deal with that in the `delegate :field_presenters`?

Reply:
I don't know. Someone thought so in all the `renders_one` blocks above, but not in the `before_render`.